include/revng/Model/model-schema.yml

```diff
@@ -937,87 +1487,76 @@ definitions:
       - name: f13_systemz
       - name: f14_systemz
       - name: f15_systemz
+
+  #
+  # Configuration
+  #
+
   - name: Configuration
     type: struct
+
+    doc: |-
+      A temporary place to put the project-local (user-local?) configuration
+      before a more comprehensive solution is introduced.
```

Comment:
Implementation detail. Demote to YAML comment.

Reply:
Done!